### PR TITLE
Add randi, randf, randf_range and randi_range blocks

### PR DESCRIPTION
**Pull request by jfcharron**
This PR allows users to use these 4 godot global functions (randi, randf, randf_range and rangi_range) to generate random numbers as requested in #185.

### CODE REVIEW

File: addons/block_code/ui/picker/categories/category_factory.gd [hunk 1]
```diff
@@ -404,6 +404,42 @@ static func get_general_blocks() -> Array[Block]:
...
+	b.block_format = "randf"
+	b.statement = "randf()"
+	b.defaults = {}
+	b.category = "Math"
+	block_list.append(b)
+
+	b = BLOCKS["parameter_block"].instantiate()
+	b.block_name = "randf_range"
+	b.variant_type = TYPE_FLOAT
+	b.block_format = "randf_range({from: FLOAT}, {to: FLOAT})"
```
Comment (manuq) [inline addons/block_code/ui/picker/categories/category_factory.gd:419]: This `block_format` goes to the UI label so you can be verbose: `Get random float from {from: FLOAT} to {to: FLOAT}`. And you can also add tooltip text with the explanation in Godot docs:
```
b.tooltip_text = "Returns a pseudo-random float between [i]from[/i] and [i]to[/i] (inclusive)."
```
The tooltip text supports rich text with [BBCode format.](https://docs.godotengine.org/en/stable/tutorials/ui/bbcode_in_richtextlabel.html)

Comment (jfcharron) [reply addons/block_code/ui/picker/categories/category_factory.gd:419]: Will do, thank you for your suggestion

Comment (manuq) [reply addons/block_code/ui/picker/categories/category_factory.gd:419]: Thanks to you for contributing this!

**Comment by jfcharron** [comment]
Sorry, about creating a new pull request (#194). I am new to this github workflow.

**Pull request closed without merging**